app/code/Magento/Catalog/Test/Mftf/Test/StorefrontOnlyXProductLeftForSimpleProductsTest.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontOnlyXProductLeftForSimpleProductsTest">
+        <annotations>
+            <features value="Catalog"/>
+            <title value="See Only * Left block"/>
+            <stories value="See Only * Left on product page if Only X left Threshold was set"/>
+            <description value="See Only * Left on product page if Only X left Threshold was set"/>
+            <severity value="MINOR"/>
+        </annotations>
+        <before>
+            <createData entity="_defaultCategory" stepKey="createCategory"/>
+            <createData entity="ApiSimpleProduct" stepKey="createProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+            <magentoCLI command="config:set {{CatalogInventoryOptionsOnlyXleftThreshold.path}} 10000" stepKey="setStockThresholdQty"/>
+            <magentoCLI command="cache:flush" stepKey="flushCache"/>
```

Comment:
Could we replace flushing the cache with clearing the config cache? Flushing the cache slows down the testing routine significantly every time it's invoked. 

Thank you.

Reply:
**Done **